code/LINDA/LINDA_fire.dm

```diff
@@ -152,18 +152,18 @@
 // Garbage collect itself by nulling reference to it
 
 /obj/effect/hotspot/proc/Kill()
-	air_master.hotspots -= src
-	DestroyTurf()
-	qdel(src)
+	PlaceInPool(src)
```

Comment:
You changed horses--you said that it would leave dangling references because of Kill() putting it on the pool...buttt when it's been mentioned that it calls Destroy(), you've changed horses to suddenly discarding it because of disliking that it calls Kill() instead of qdel()? 

Correct me if I'm wrong, but air_master.hotsposts -= src being under Destroy() instead of Kill() would be more robust than under Kill(), as anything that qdel()'s a hotspot won't remove it from the air_master--if I'm not mistaken.